### PR TITLE
1339 Add margin to listbox button checkmark

### LINKED ISSUE #1339: Listbox-Button: insufficient whitespace next to selection icon when text is long
This issue came up as part of https://github.com/eBay/ebayui-core/pull/1310

Screen Shot 2021-01-26 at 2 14 07 PM

### PR DESCRIPTION
**Pull request by austingardner**
## Description
Added margin to listbox to ensure checkmark isn't obscured by text.
Added example with long text to demo the added margin

## References
closes #1339 

## Screenshots
Screen Shot 2021-02-09 at 3 11 36 PM

### CODE REVIEW

**Review by agliga — CHANGES_REQUESTED** [review]
Actually looking at it this change is not needed.
What it looks like is we need to add the `margin-left: 8px`  in `listbox__option`

This issue came from ebayui not using `listbox-button__option`, however this will show up in `listbox__option` so we should change it there. 

Screen Shot 2021-02-09 at 11 50 09 AM

**Comment by austingardner** [comment]
Okay I removed the changes from listbox-button and applied them to listbox... let me know if this is what you were looking for

File: docs/_includes/common/listbox-button.html [hunk 1]
```diff
@@ -305,3 +305,5 @@ <h3 id="listbox-button-fluid">Fluid</h3>
 </span>
     {% endhighlight %}
 </div>
+
+
```
Comment (agliga) [inline docs/_includes/common/listbox-button.html:309]: Remove whitespace

**Review by agliga — COMMENTED** [review]
LGTM other than removing whitespace. will merge with that.

**Review by agliga — APPROVED** [review]

File: docs/_includes/common/listbox.html [hunk 2]
```diff
@@ -114,4 +114,57 @@ <h3 id="listbox-selected">Selected State</h3>
     </div>
 </div>
     {% endhighlight %}
+    <h3 id="listbox-selected">Long Option</h3>
```
Comment (ianmcburnie) [inline docs/_includes/common/listbox.html:117]: This should be a test case / edge case under storybook only. No need for it to be on the main docs.
